service/matching/matchingEngine.go

```diff
@@ -449,6 +453,30 @@ func (e *matchingEngineImpl) CancelOutstandingPoll(ctx context.Context, request
 	return nil
 }
 
+func (e *matchingEngineImpl) DescribeTaskList(ctx context.Context, request *m.DescribeTaskListRequest) (*workflow.DescribeTaskListResponse, error) {
+	domainID := request.GetDomainUUID()
+	taskListType := persistence.TaskListTypeDecision
+	if request.DescRequest.GetTaskListType() == workflow.TaskListTypeActivity {
+		taskListType = persistence.TaskListTypeActivity
+	}
+	taskListName := request.DescRequest.TaskList.GetName()
+
+	taskList := newTaskListID(domainID, taskListName, taskListType)
+	tlMgr, err := e.getTaskListManager(taskList)
+	if err != nil {
+		return nil, err
+	}
+
+	pollers := []*workflow.PollerInfo{}
+	for _, poller := range tlMgr.GetAllPollerInfo() {
+		pollers = append(pollers, &workflow.PollerInfo{
+			Identity:  common.StringPtr(poller.identity),
+			Timestamp: common.TimePtr(poller.timestamp),
```

Comment:
this TimePtr is misleading, it is still int64 pointer.

Reply:
cannot use Int64Ptr since golang does not support overloading, use TimeInt64Ptr